include/color_spinor_field_order.h

```diff
@@ -476,12 +476,12 @@ namespace quda
       complex<storeFloat> *v;        /**< Field memory address this wrapper encompasses */
       const int idx;                 /**< Index into field */
     private:
-      const Float scale;             /**< Float to fixed-point scale factor */
-      const Float scale_inv;         /**< Fixed-point to float scale factor */
+      const Float scale;     /**< Float to fixed-point scale factor */
+      const Float scale_inv; /**< Fixed-point to float scale factor */
```

Comment:
There seem to be formatting changes to lines here that haven't been edited.  This can cause unnecessary conflicts.

Reply:
I have hopefully undone that now.